.github/ISSUE_TEMPLATE/1-bug.yml

```
@@ -16,82 +17,86 @@ body:
           required: true
-        - label: I searched [issues](https://github.com/search?q=user%3Asyntax-tree&type=Issues) and couldn’t find anything (or linked relevant results below)
+        - label: I searched [issues](https://github.com/search?q=user%3Asyntax-tree&type=issues) and [discussions](https://github.com/search?q=user%3Asyntax-tree&type=discussions) and couldn’t find anything (or linked relevant results below)
           required: true
     id: initial-checklist
     type: checkboxes
   - attributes:
       description: |
-        Please test using the latest version of the relevant packages to make
-        sure your issue has not already been fixed.
-      label: Affected packages and versions
+        Which packages cause the bug?
+        Please test using latest versions.
+        Example: `some-package@1.2.3`.
+      label: Affected package
     id: affected-versions
     type: input
     validations:
       required: true
-  - attributes:
-      description: |
-        Link to repository or sandbox with runnable example of the issue.
-        Alternatively use the next section *Steps to reproduce*.
-      label: Link to runnable example
-    id: runnable-example
-    type: input
-    validations:
-      required: false
   - attributes:
       description: |
         How did this happen?
-        Please provide a [minimal and reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+
+        Provide a [minimal and reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+        Link to a repository or sandbox or show the code here.
+        Remove everything that is not needed.
+        Use plain Node and npm if possible.
       label: Steps to reproduce
     id: steps-to-reproduce
     type: textarea
     validations:
       required: true
   - attributes:
-      description: What should happen?
-      label: Expected behavior
-    id: expected-behavior
+      description: |
+        What happens?
+        What is wrong with the current behavior?
+      label: Actual behavior
+    id: actual-behavior
     type: textarea
     validations:
       required: true
   - attributes:
-      description: What happens instead?
-      label: Actual behavior
-    id: actual-behavior
+      description: |
+        What should happen instead?
+        Why should that happen?
+        Why is that better than the current behavior?
+      label: Expected behavior
+    id: expected-behavior
     type: textarea
     validations:
       required: true
   - attributes:
-      label: Affected runtime and version
+      label: Runtime
       description: |
-        What runtime are you using (such as `node@18.2` or `deno@1.12.2`)?
-        Please test using the latest version.
+        What runtime are you using?
+        Please test using latest versions.
+        Example: `node@23.2.0` or `deno@2.0.6`.
```

Maybe add a browser as an example? If the error happens in a browser, the Node.js version used to launch a dev server is irrelevant. I think this is a common cause of confusion.

```suggestion
        Example: `node@23.2.0`, `deno@2.0.6`, or `Google Chrome 131`
```

---

Maybe maybe.

I added a note somewhere else to prefer reproducing the problem with plain Node/npm.

The thinking goes that sometimes people provide a repo with an entire Next app, when it’s a question about how markdown compiles to HTML.

Most of our stuff works anywhere, browsers, deno/bun/node. And node is easiest for us to debug. It’s also what we base our compatibility on.

I thought about adding Bun too. But I preferred being less verbose.

We could also drop it, and do:

```suggestion
        Example: `node@23.2.0`.
```

I also tried to use the actual output of version commands right now. So, looking at my Chrome’s about info, perhaps this should be `Google Chrome 131.0.6778.86`?

I think I’d prefer my above suggestion (just node) or:

```suggestion
        Example: `node@23.2.0` or `Chrome 131.0.6778.86`.
```

---

I agree with all of that, but in practice people also run into issues in other runtimes and not everyone is equally able or wanting to make a nice Node.js reproduction. Also some packages work differently in the browser or Node.js, i.e. when we specify the `browser` export condition.

---

right, true! any of these alternatives of mine that you 👍

---

Either is :+1: really, but I prefer to include Chrome.